<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BendingStressWidget</class>
 <widget class="QFrame" name="BendingStressWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {background-color: #ffffee}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="moi_label">
     <property name="text">
      <string>I</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="c_bottom_units">
     <property name="text">
      <string>cm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="mom_of_inertia"/>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="c_bottom"/>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="ResultsWidget" name="results">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Results</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="c_top_label">
     <property name="text">
      <string>c&lt;sub&gt;top&lt;/sub&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="MOE_units">
     <property name="text">
      <string>MPA</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="c_bottom_label">
     <property name="text">
      <string>c&lt;sub&gt;bottom&lt;/sub&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="moi_units">
     <property name="text">
      <string>cm&lt;sup&gt;4&lt;/sup&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="MOE_label">
     <property name="text">
      <string>MOE</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="c_top_units">
     <property name="text">
      <string>cm</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="MOE"/>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="c_top"/>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QLabel" name="force_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>force</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="moment_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>moment</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="moment">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="moment_units">
       <property name="text">
        <string>kN.m</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <widget class="QPushButton" name="compute_btn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>compute</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="force">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="force_units">
       <property name="text">
        <string>kN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="area_label">
     <property name="text">
      <string>A</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="area"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="area_units">
     <property name="text">
      <string>cm&lt;sup&gt;2&lt;/sup&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResultsWidget</class>
   <extends>QGroupBox</extends>
   <header>resultswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>MOE</tabstop>
  <tabstop>area</tabstop>
  <tabstop>mom_of_inertia</tabstop>
  <tabstop>c_top</tabstop>
  <tabstop>c_bottom</tabstop>
  <tabstop>compute_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
